### PR TITLE
Add: Morning style variation.

### PR DESCRIPTION
**Pull request by juanfra**
**Description**

Add the Morning (Literata 60pt + Ysabeau Office) global style variation.
Partial for https://github.com/WordPress/twentytwentyfive/issues/193

**Known limitations**

- Many font sizes don't match the presets. They were hardcoded in this PR.
- The post author section has a hardcoded font preset in the template that doesn't match the design. So the author size is different than the text that says "Written by".
- The font size for the comments section looks small in the context, but there's no design for the comments for this variation.

**Screenshots**

Screenshot 2024-09-09 at 18-30-53 Style Variations – Stories®

**Testing Instructions**

1. Go to Appearance > Editor > Styles
2. At the top of the styles panel, select "Morning".
3. Save.
4. Compare the design with the design in Figma

### CODE REVIEW

**Comment by beafialho** [comment]
@juanfra I noticed the titles are looking different than the headings, they seem to have specific styling. If you disable them, the appearance looks right:

https://github.com/user-attachments/assets/32529e43-04b9-41ca-aa00-f492fc63a5d3

The font sizes are not looking very good. I think we should reset all the heading sizes in this variation to use the sizes from the default one, as they look much better.

**Review by carolinan — CHANGES_REQUESTED** [review]
### Style 2
Headings use the text color `#191919 `(contrast) against the background `#182949` (accent 3). This has a contrast ratio of
1.21:1. It needs to be 4.5:1.

I believe this is meant to use base, to match the paragraph text color.

**Comment by juanfra** [comment]
Thanks, Carolina. Good catch. I've updated the variation with that change.

File: styles/morning.json [hunk 1]
```diff
@@ -0,0 +1,338 @@
...
+			"link": {
+				"color": {
+					"text": "var:preset|color|contrast"
+				}
+			}
+		},
+		"variations": {
+			"post-terms-1": {
+				"typography": {
+					"fontSize": "16px"
```
Comment (carolinan) [inline styles/morning.json:244]: Does this need to be hardcoded? Is there no matching responsive preset size?

**Comment by carolinan** [comment]
Feedback: I am struggling to read font size preset `small` which is 14px, up to preset `medium` 18px with this font over the grey background.
The text color and background color has a contrast ratio of 4.66:1 which is within the WCAG AA level requirement, but for me, a higher contrast or even better a larger size would help.

This is even more appearant with the new zoom out.

Both the default and Dusk are much more readable to me because of the different size of these fonts.

File: styles/morning.json (hunk 1 above)
Comment (juanfra) [reply styles/morning.json:244]: There's no preset for 16px, which is the size of the font size for this particular element in the variation. We have `--wp--preset--font-size--normal` from Gutenberg, which is 16px, but we can't use that.

If we don't want to hardcode the value, we could use medium (18px) or small (14px) alternatives.

**Comment by carolinan** [comment]
https://github.com/user-attachments/assets/4e86f740-679d-4011-a521-fe750ff3be7c

**Comment by carolinan** [comment]
You can see just how much smaller this font is by the number of menu items that fit on the same row.

**Comment by juanfra** [comment]
Thanks for the feedback, Carolina.

The issues you highlighted are related to the size of the Ysabeau font, compared to fonts used in other variations. The menu item size will happen the other way around with variations like Twilight or Midnight, where the font size for those elements is bigger.

I believe this is a conversation we want to have with design, as some of these elements are probably intended to be like that. Given the time constraints of beta 1, if these are the only blockers, we could merge this and have a new issue to discuss what you highlighted and get @beafialho and @jasmussen in the conversation.

File: styles/morning.json [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+				},
+				"typography": {
+					"fontSize": "1.5rem",
+					"fontWeight": "500",
+					"letterSpacing": "-0.18px"
+				},
+				"elements": {
+					"cite": {
+						"typography": {
+							"fontSize": "1.12rem",
```
Comment (carolinan) [inline styles/morning.json]: I believe this needs to use em if it is supposed to be relative to the size of the quote.

File: styles/morning.json (hunk 1 above)
Comment (carolinan) [reply styles/morning.json:244]: I'd always go for the larger :)

**Comment by jasmussen** [comment]
Zoom out, give or take any changes that comes to this, should not dictate any font size rules in the theme. 

The core of the issue, though: is the gray on light gray font legible at 14px? For me it is, so I'd be tempted to echo Juan and merge this for Beta 1 and then revisit the font sizes (or perhaps the font _weights_) after the beta period. Notably, Bea is AFK this week, a vacation planned months before the 6.7 cycle even had dates attached, and I'd appreciate her quick gut-check when she gets back.

How does that sound?

**Comment by carolinan** [comment]
Nobody is pressuring anyone to not have a vacation ❤️

**Comment by jasmussen** [comment]
> Nobody is pressuring anyone to not have a vacation ❤️

Oh, I didn't mean to imply otherwise, and I supremely agree!

File: styles/morning.json (hunk 2 above)
Comment (juanfra) [reply styles/morning.json]: The intention was to have it in `rem` to match the design (~18px). Now that I see this, we can use the medium preset. Unless you prefer to have the `em` value.

Screenshot 2024-09-24 at 11 24 56

File: styles/morning.json (hunk 1 above)
Comment (juanfra) [reply styles/morning.json:244]: Should we run this by Bea once she's back?

**Comment by juanfra** [comment]
> How does that sound?

That sounds great to me. 

I'd prefer not to iterate on the designs now and have Bea (after she's back from her well-deserved time off) participate in the discussion, as there might be some reasoning behind some of the selections (sizes, colors, etc.). We should keep track of all these concerns so we can return to them.

**Comment by carolinan** [comment]
Please remember that the beta 1 date is not something we have the power to move.

**Pull request merged**